Cargo.toml

```diff
@@ -56,8 +57,7 @@ fiat-crypto = { version = "0.1.6", optional = true}
 
 [features]
 nightly = ["subtle/nightly"]
```

Comment:
Unrelated, but I thought `nightly` was removed at some point? The flag does nothing since subtle v2.2.

Reply:
nvm that's in my README PR

Reply:
Can remove it in a followup.